sdk/storage/CMakeLists.txt

```diff
@@ -9,19 +9,37 @@ set(CMAKE_CXX_STANDARD_REQUIRED True)
 
 option(BUILD_STORAGE_SAMPLES "Build sample codes" ON)
 
-set(AZURE_STORAGE_HEADER
+set (AZURE_STORAGE_BLOB_HEADER
     inc/blobs/blob.hpp
     inc/blobs/internal/protocol/blob_rest_client.hpp
     inc/common/internal/protocol/rest_client_utility.hpp
     inc/common/xml_wrapper.hpp
 )
 
-set(AZURE_STORAGE_SOURCE
+set (AZURE_STORAGE_ADLS_HEADER
+    inc/common/deps/json.hpp
```

Comment:
maybe something like `inc/external/json.hpp` looks better? i don't know

Reply:
vcpkg has nlohmann-json port, do we have to include third-party dependency in our code base?

Reply:
I'm fine with both. We can let architecture board decide which one should be used.